flottform/forms/src/create-flottform-input.ts

```diff
@@ -217,9 +218,31 @@ export function createFlottformInput({
 			createChannelElement.querySelector<HTMLElement>('.flottform-button')!;
 		createChannelButton.style.background = `conic-gradient(#7EA4FF ${Math.round((currentSize / totalSize) * 100)}%, #FFF ${Math.round((currentSize / totalSize) * 100)}%`;
 	},
-	onResult = (files) => {
-		if (inputField) {
-			inputField.files = files;
+	onResult = async (
+		inputField,
+		incoming: { fileMeta: FileMetaInfos; arrayBuffers: Array<ArrayBuffer> }
+	) => {
+		if (!inputField) {
+			logger.warn('no input field, no need to run onResult');
+			return;
+		}
+		if (incoming.fileMeta.data === 'input:file' || incoming.fileMeta.data === 'canvas:png') {
+			const fileName = incoming.fileMeta.name ?? 'no-name';
+			const fileType = incoming.fileMeta.type ?? 'application/octet-stream';
+
+			const fileForForm = new File(incoming.arrayBuffers, fileName, { type: fileType });
+			const dt = new DataTransfer();
+			dt.items.add(fileForForm);
+			inputField.files = dt.files;
+		}
+		if (incoming.fileMeta.data === 'input:text') {
+			const decoder = new TextDecoder();
+			let result = '';
+			for (const arrayBuffer of incoming.arrayBuffers) {
+				// TODO Test whether umlaute work on fragmented text - seem to work well
```

Comment:
Let's add tests for this (playwright) to ensure this is easy to check in the future as well.

Should be a new task and remove this comment then after issue is created